aiida_common_workflows/cli/launch.py

```diff
@@ -103,3 +105,102 @@ def cmd_relax(
 
     builder = generator.get_builder(structure, engines, protocol, relaxation_type, threshold_forces, threshold_stress)
     utils.launch_process(builder, daemon)
+
+
+@cmd_launch.command('eos')
+@click.argument('plugin', type=types.LazyChoice(functools.partial(get_workflow_entry_point_names, 'relax', True)))
+@options.STRUCTURE(help='The structure to relax.')
+@options.PROTOCOL(type=click.Choice(['fast', 'moderate', 'precise']), default='fast')
+@options.THRESHOLD_FORCES()
+@options.THRESHOLD_STRESS()
+@options.DAEMON()
+@click.option('--show-engines', is_flag=True, help='Show information on the required calculation engines.')
+def cmd_eos(plugin, structure, protocol, threshold_forces, threshold_stress, daemon, show_engines):
+    """Relax a crystal structure using the common relax workflow for one of the existing plugin implementations.
+
+    The command will automatically try to find and load the codes that are required by the plugin workflow. If no code
+    is installed for at least one of the calculation engines, the command will fail. Use the `--show-engine` flag to
+    display the required calculation engines for the selected plugin workflow.
+    """
```

Comment:
The docstring is copied from relax. It should be modified as it appears when the users does not specify the code

Reply:
Actually, I don't quite get your second sentence. Could you please clarify?

Reply:
Ah no, sorry. It appears when invoking `-h`. Got confused.